src/components/ui/Label.tsx

```diff
@@ -4,25 +4,61 @@ import { pixelWhiteBorderStyle } from "features/game/lib/style";
 
 interface Props {
   className?: string;
+  type?: string;
 }
-export const Label: React.FC<Props> = ({ children, className }) => {
+export const Label: React.FC<Props> = ({ children, className, type }) => {
```

Comment:
How about you set a default value for `type` so you don't have to pass the `default` prop?

`React.FC<Props> = ({ children, className, type = "default" })`

Reply:
I have doubts about this decision because I think the default value should be used for items that are used frequently, while we use the "info" (blue) value more than the "default" (silver) itself, or I think we can swap default for the blue one then create another type for the silver one?